internal/server/data/data.go

```diff
@@ -116,11 +120,34 @@ func get[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (*T, error)
 	return result, nil
 }
 
+func setOrg(db *gorm.DB, model any) {
+	member, ok := model.(orgMember)
+	if !ok {
+		return
+	}
+
+	org := OrgFromContext(db.Statement.Context)
+	member.SetOrganizationID(org.ID)
+}
+
+type orgMember interface {
+	IsOrganizationMember()
+	SetOrganizationID(id uid.ID)
+}
```

Comment:
Interfaces in Go should be defined where they are used, not where they are implemented. This is definitely the right place for this interface because it is used in `data`.  This practices aligns nicely with the philosophy of "accept interfaces, return structs".